Client/Frontend/Home/TwoLineCell.swift

```diff
@@ -4,6 +4,111 @@
 
 import UIKit
 
+class ThumbnailCell : UICollectionViewCell {
```

Comment:
Why is this in `TwoLineCell.swift`? It's not related to `TwoLineCell`. Put this in its own file?